### ISSUE TITLE
Adding custom dataloaders, based on recipes

### ISSUE THREAD

**Issue by ShpihanVlad** [Deci-AI/super-gradients#1663]
### 💡 Your Question

Hi, I have a question about integrating user-made recipes into dataloaders. Basically, I want to make something like:
```python
@register_dataloader("my_train_dataloader")
def my_train_dataloader(dataset_params: Dict = None, dataloader_params: Dict = None) -> DataLoader:
    return get_data_loader(
        config_name="my_dataset_dataset_params",
        dataset_cls=YoloDarknetFormatDetectionDataset,
        train=True,
        dataset_params=dataset_params,
        dataloader_params=dataloader_params,
    )
```
I have a recipes folder with dataset_params subfolder, the same format as used for pre-defined recipes. How can I configure my dataloaders to access my recipes, not pre-defined ones? I want for them to be usable inside notebooks, so `@hydra.main` is not accessible for launching.
Thanks!

**Comment by shaydeci** [comment]
@ShpihanVlad if you are using coded training, you can simply instantiate the dataloaders as with any regular Pytorch training.
Then pass them through train_dataloader and val_dataloader arguments in Trainer.train(). No need to define configuration files for them.
